source/baseObject.py

```diff
@@ -283,6 +282,12 @@ def getScript(self,gesture):
 				return self._gestureMap[identifier].__get__(self, self.__class__)
 			except KeyError:
 				continue
+			except AttributeError:
+				log.exception((
+					"Did you miss to initialize the base class?"

```

Comment:
This reads slightly strangely to me 😄 How about:
```suggestion
					"Missing base class initialization?"
```

Reply:
Strange to see a question targeting the developer in the log.
```suggestion
					"Maybe the base class was not initialized."

```

Reply:
> > I guess this does not fit well in the existing change log sections, as the bug it fixes mostly affects developers.
> 
> Therefore, I think changes for developers fits.

I directly edited the PR description for clarity with a proposed change log entry.
Feel free to amend as you see fit.